### PR TITLE
New features introduced

### PR DESCRIPTION
**Pull request by preciousuwhubetine**
This branch introduces the following features:
- Add a new task by entering the task's name in the input and hitting `enter` on the keyboard or clicking the `return` button.
- Edit a task by clicking the vertical ellipsis to the right of the task's description, entering a new task description, and hitting the `enter` button on the keyboard.
- A task can be deleted by clicking the vertical ellipsis to the right of the task's description and then clicking the delete button.

### CODE REVIEW

**Comment by TedLivist** [comment]
## STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @preciousbetine,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### I think you submitted the wrong form. Your code and your PR description describe add and remove project but you submitted interactive list project form. Kindly look into that.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by preciousuwhubetine** [comment]
Oh. My bad! Thanks @TedLivist

File: src/tasks.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+const updateLocalStorage = () => {
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+};
+
+const addNewTask = (input) => {
+  if (input.value.trim().length === 0) return;
+  tasks.push({
+    description: input.value.trim(),
+    completed: false,
+    index: tasks.length,
```
Comment (levy002) [inline src/tasks.js]: - [x] Kindly make sure the `index` of the task always starts at `1`. It means if we added the fifth task then it's `index` must be `5`. 

     **_Hint: index: `tasks.length + 1`_**

File: src/tasks.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  updateLocalStorage();
+};
+
+const removeTask = (index) => {
+  tasks = tasks.filter((task) => task.index !== index);
+
+  // Update remaining task indices
+  tasks.forEach((task, idx) => {
+    task.index = idx;
+  });
```
Comment (levy002) [inline src/tasks.js]: - [x] KIndly make sure to update task indexes and make sure it always starts at `1`.

  _**Hint:  `task.index = idx + 1`**_

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @preciousbetine   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Highlights
- Professional PR :heavy_check_mark: 
- Professional README :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @preciousbetine 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
  - All changes requested have been implemented 👏🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**